README.md

```diff
@@ -46,25 +46,40 @@ e.g `sudo echo "192.168.122.235 ostest-api.test.metalkube.org" >> /etc/hosts`.
 Then you can interact with the k8s API on the bootstrap VM e.g
 `sudo oc status --verbose --config /etc/kubernetes/kubeconfig`.
 
-You can also use the ironic by talking to its API with the openstack client (from the
+You can also see the status of the bootkube.sh script which is running via
+`journalctl -b -f -u bootkube.service`.
+
+## 2) Interact with Ironic from the host
+
+You can test ironic by talking to its API with the openstack client (from the
 virt host)
-`export OS_TOKEN=fake-token`
-`export OS_URL=http://ostest-api.test.metalkube.org:6385/`
 
-To Define a NODE (Get \<MAC\> and \<IPMI\_PORT\> from instackenv.json
-`NODE_UUID=$(openstack baremetal node create --driver ipmi --driver-info ipmi_address=192.168.122.1 --driver-info ipmi_port=6230 --driver-info ipmi_username=admin --driver-info ipmi_password=password --driver-info deploy_kernel=http://172.22.0.1/images/tinyipa-stable-rocky.vmlinuz --driver-info deploy_ramdisk=http://172.22.0.1/images/tinyipa-stable-rocky.gz -f value -c uuid)`
-`openstack baremetal port create 00:32:49:d0:63:75 --node $NODE_UUID`
-`openstack baremetal node set $NODE_UUID --instance-info image_source=https://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img --instance-info image_checksum=f8ab98ff5e73ebab884d80c9dc9c7290 --instance-info root_gb=10  --property root_device='{"name": "/dev/vda"}'`
-`openstack baremetal node manage $NODE_UUID --wait`
-`openstack baremetal node provide $NODE_UUID --wait`
+```
+export OS_TOKEN=fake-token
+export OS_URL=http://ostest-api.test.metalkube.org:6385/
+```
 
-And to deploy a node with the cirros image (image\_source above)
-`openstack baremetal node deploy $NODE_UUID`
+To define the master nodes, you can use:
 
-You can also see the status of the bootkube.sh script which is running via
-`journalctl -b -f -u bootkube.service`.
+```
+openstack baremetal create ocp/master_nodes.json
+```
+
+Then to deploy you can do e.g:
+
+```
+openstack baremetal node set $NODE_UUID --instance-info image_source=https://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img --instance-info image_checksum=f8ab98ff5e73ebab884d80c9dc9c7290 --instance-info root_gb=10  --property root_device='{"name": "/dev/vda"}'
+openstack baremetal node manage $NODE_UUID --wait
```

Comment:
Strictly speaking, the `manage` action should go before populating instance_info. It does not matter, just looks less awkward to me :)

Also, may I advertise you `metalsmith` CLI as a means of testing standalone ironic? :) It's pretty neat.

Reply:
Ack yeah I didn't change the order here, only fix the code block formatting - happy to switch it and/or move to metalsmith in a follow-up :)

When we've got this working I hope we'll have an init container which drives the ironic API, so we can transparently switch in whatever tool/code is preferable.